src/sharing/ShareToPm.js

```diff
@@ -162,12 +162,7 @@ class ShareToPm extends React.Component<Props, State> {
     let sharePreview = null;
     if (sharedData.type === 'image') {
       sharePreview = (
-        <Image
-          source={{ uri: sharedData.sharedImageUrl }}
-          width={200}
-          height={200}
-          style={styles.imagePreview}
-        />
```

Comment:
Worth a quick mention in the commit message that this isn't a change in Flow -- these props are there in the type in RN at our current version, and it's that type definition in RN that's getting stricter in the upgrade, to match the docs as you describe.

Here's the relevant bit of the type definition as of our current v0.62.x version:
```js
  /**
   * See https://facebook.github.io/react-native/docs/image.html#style
   */
  style?: ?ImageStyleProp,

  // Can be set via props or style, for now
  height?: ?DimensionValue,
  width?: ?DimensionValue,
```